pkg/k8shandler/desirednodestate.go

```diff
@@ -478,6 +487,38 @@ func (cfg *desiredNodeState) getESContainer() v1.Container {
 	}
 }
 
+func (cfg *desiredNodeState) getProxyContainer() v1.Container {
+	// probe := getReadinessProbe()
+	return v1.Container{
+		Name:            "rbac-proxy",
+		Image:           cfg.ProxyImage,
+		ImagePullPolicy: "IfNotPresent",
+		// Env:             cfg.getEnvVars(),
+		Ports: []v1.ContainerPort{
+			v1.ContainerPort{
+				Name:          "rbac-proxy",
+				ContainerPort: 60000,
+				Protocol:      v1.ProtocolTCP,
+			},
+		},
+		// ReadinessProbe: &probe,
+		VolumeMounts: []v1.VolumeMount{
+			v1.VolumeMount{
+				Name:      fmt.Sprintf("%s-%s", cfg.ClusterName, "metrics-volume"),
+				MountPath: "/etc/rbac-proxy/secrets",
+			},
+		},
+		Args: []string{
+			"--tls-cert-file=/etc/rbac-proxy/secrets/tls.crt",
+			"--tls-private-key-file=/etc/rbac-proxy/secrets/tls.key",
+			"--tls-private-key-file=http://localhost:9200",
```

Comment:
`--tls-private-key-file` is duplicated where one is actually a file and the other is a url.  should it be duped?

Reply:
Yes, that was a mistake